### ISSUE TITLE
Rename "firebase" export

### ISSUE THREAD

**Issue by mrshll** [prescottprue/redux-firebasev3#17]
The connect export is currently named 'firebase'. It would be great to call this `firebaseConnect` so that the normal firebase library can be accessed when needed.

Right now I do this to avoid conflicts:

```
import ReduxReactFirebase from 'redux-firebasev3';
const firebaseCompose = ReduxReactFirebase.firebase;
```

**Comment by prescottprue** [comment]
@mmoutenot Great suggestion! Love input like this.

I have actually run into this myself while still using `redux-react-firebase`. Going to look into the change today and post any updates.

**Comment by Bojhan** [comment]
Yup this would be great, now I am doing a few hacks to get acces to Firebase functions. Mostly just the debugging one :)

**Comment by mrshll** [comment]
Worth noting that this is definitely a breaking change that people will have to be careful of when updating.

**Comment by prescottprue** [comment]
@mmoutenot I totally agree on breaking changes.

Hopefully the #18 code that is in v0.1.3 makes this not as necessary for now, but I still like the idea of coming up with another name.

**Comment by prescottprue** [comment]
@mmoutenot To take care of the breaking changes problem you mentioned, in `v0.1.5` the connect function will be exposed to both `firebase` and `firebaseConnect`.

The [API Docs](https://prescottprue.gitbooks.io/redux-firebasev3/content/api.html) are updated too.

**Comment by mrshll** [comment]
YEAH! Awesome :)

**Comment by prescottprue** [comment]
Changes included in [`v0.1.5` release](https://github.com/prescottprue/redux-firebasev3/releases/tag/v0.1.5).
